build.gradle.kts

```
@@ -16,13 +16,16 @@
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.gradle.api.tasks.testing.logging.TestLogEvent
 
-group = "org.mongodb"
+group = "org.mongodb.kbson"
 
 version = "1.0-SNAPSHOT"
```

```suggestion
version = "1.0.0-SNAPSHOT"
```

So it matches full semantic versioning

---

Done - good catch